src/components/dataDisplays/SightingsDisplay.jsx

```diff
@@ -32,58 +30,56 @@ export default function SightingsDisplay({ sightings, loading }) {
 
   const columns = [
     {
-      name: 'id',
-      label: intl.formatMessage({
-        id: 'SIGHTING_ID',
-      }),
+      name: 'time',
+      labelId: 'SIGHTING_TIME',
       options: {
-        customBodyRender: value => (
-          <Link href={`/sightings/${value}`}>{value}</Link>
-        ),
+        cellRenderer: cellRendererTypes.specifiedTime,
       },
     },
     {
-      name: 'individuals',
-      label: intl.formatMessage({ id: 'INDIVIDUALS' }),
-      options: {
-        customBodyRender: individuals => (
-          <span>
-            {individuals.map((id, i) => (
-              <span key={id}>
-                {i !== 0 && ', '}
-                <Link href={`/individuals/${id}`}>{id}</Link>
-              </span>
-            ))}
-          </span>
-        ),
-      },
+      name: 'locationId_value',
+      labelId: 'REGION',
+      align: 'left',
```

Comment:
There's a locationRenderer; don't know if you are intentionally avoiding it here.

Reply:
Yeah, locationRenderer requires more resources to derive the region label from the region guid. Since the backend is just sending us the label no reason to convert the guid